vite.config.js

```diff
@@ -1,54 +1,39 @@
 import { defineConfig } from 'vite';
 import { svelte } from '@sveltejs/vite-plugin-svelte';
-import { createHtmlPlugin } from 'vite-plugin-html';
+import fs from 'fs';
+import path from 'path';
 
+const htmlVars = {
+  browser_lang: 'de',
+  page_title: 'Generiere dein Datenauskunftsbegehren | Digitale Gesellschaft',
+  description: 'Gemäss Datenschutzgesetz hat jede Person das Recht zu erfahren, welche Daten über sie gespeichert sind, und diese – wenn nötig – löschen oder korrigieren zu lassen.',
+  description_short: 'Gemäss Datenschutzgesetz hat jede Person das Recht zu erfahren, welche Daten über sie gespeichert sind',
+  browser_outdated: '<p>Du verwendest einen alten Browser.<br>Der Generator funktioniert damit leider nicht.</p><p>Bitte verwende eine aktuelle Version des Browsers deiner Wahl.</p>'
+};
 
 export default defineConfig({
   plugins: [
-    // commonjs(),
     svelte(),
-    createHtmlPlugin({
-      minify: true,
-      entry: 'src/main.js',
-      template: 'index.html',
-      inject: {
-        data: {
-          // @TODO Redaktion description_short: Shorten description for SEO purposes or leave the long version?
-          browser_lang: 'de',
-          page_title: 'Generiere dein Datenauskunftsbegehren | Digitale Gesellschaft',
-          description: 'Gemäss Datenschutzgesetz hat jede Person das Recht zu erfahren, welche Daten über sie gespeichert sind, und diese – wenn nötig – löschen oder korrigieren zu lassen.',
-          description_short: 'Gemäss Datenschutzgesetz hat jede Person das Recht zu erfahren, welche Daten über sie gespeichert sind',
-          browser_outdated: '<p>Du verwendest einen alten Browser. <br>Der Generator funktioniert damit leider nicht.</p><p>Bitte verwende eine aktuelle Version des Browsers deiner Wahl.</p>'
-        },
-      },
-    })
+    {
+      name: 'html-template-vars',
+      transformIndexHtml(html) {
+        // einfache Platzhalter-Ersetzung: %varname%
+        return html.replace(/%(\w+)%/g, (_, key) => htmlVars[key] ?? '');
+      }
+    }
   ],
-
   server: {
     open: true
   },
   preview: {
     open: true,
     port: 8080,
   },
-  optimizeDeps: {
-    include: ['lodash-es'],
-  },
   build: {
     outDir: 'dist',
     emptyOutDir: true,
-    rollupOptions: {
-      input: 'src/main.js',
-      output: {
-        entryFileNames: 'build/[name]-[hash].js',
-        chunkFileNames: 'build/[name]-[hash].js',
-        assetFileNames: 'build/[name]-[hash].[ext]',
-      },
-      sourcemap: true,
-      name: 'app',
-    },
+    sourcemap: true
```

Comment:
Müssen wür aufpassen dass wir das nur in development machen. für prod build wollen wir nicht den gesamten source code miteinbinden. Mal zu testen...

Reply:
Habe es absichtlich auf true gelassen. Es werden .map Dateien generiert. Diese werden vom Browser by default nicht geladen wenn man die Applikation normal nutzt. Erst wenn es Fehler in der Konsole gibt und man draufklickt lädt er die Dateien. Da das Projekt sowieso OpenSource ist sehe ich kein Problem den Quellcode "bereitzustellen". Ist aber nützlich wenn man Fehler etwas anschauen will auf Prod. Können es aber auch deaktivieren wenn es dich/euch stört.